File: pset6/mario.py
from cs50 import get_int

def main():
    while True:
        height = get_int("What is the Height of pyramid?   ")
        if height >= 1 and height <= 8:
            break
    create_pyramids(height)


def create_pyramids(h):
    dm = h;
    for i in range(1, h + 1):
        for j in range(1, h + 1):
            if (j>=dm):
                print("#", end="")
            else:
                print(" ", end="")

        print(" ", end="")
        print(" ", end="")
        k=h
        while k>=dm:
            print("#", end ="")
            k = k - 1
        print()
        dm = dm -1








if __name__=='__main__':
    main()

File: pset1/mario.c
#include <stdio.h>
#include <cs50.h>

void create_pyramids(int);

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height > 8 || height < 1);

    create_pyramids(height);
}

void create_pyramids(int h)
{
    int dm = h;
    for (int i = 1; i <= h; i++)
    {
        for (int j = 1; j <= h; j++)
        {
            if (j >= dm)
            {
                printf("#");
            }
            else
            {
                printf(" ");
            }
        }

        printf("  ");

        int k = h;
        while (k >= dm)
        {
            printf("#");
            k--;
        }
        printf("\n");
        dm--;
    }
}